packages/components/src/test/properties/UniquePropertyValuesSelector.test.tsx

```diff
@@ -294,6 +294,86 @@ describe("UniquePropertyValuesSelector", () => {
+      sinon.stub(Presentation.presentation, "getPagedDistinctValues").resolves({
+        total: 1,
+        items: [{ displayValue: "1410-07-15", groupedRawValues: [""] }],
+      });
+      const datePropertyDescription = {
+        name: "#propertyName",
+        displayLabel: "property",
+        typename: "shortDate",
+        editor: undefined,
+      };
+
+      const { queryByText, getByText, user } = render(
+        <UniquePropertyValuesSelector property={datePropertyDescription} onChange={() => {}} imodel={testImodel} descriptor={descriptor} />,
+      );
+
+      // open menu
+      const selector = await waitFor(() => getByText("unique-values-property-editor.select-values"));
+      await user.click(selector);
+
+      // assert that row is displayed correctly
+      await waitFor(() => {
+        expect(queryByText(new Date("1410-07-15").toLocaleDateString())).to.not.be.null;
+      });
+    });
+
+    it(`displays empty value string when typename is 'dateTime' but date is set as empty string`, async () => {
+      sinon.stub(Presentation.presentation, "getPagedDistinctValues").resolves({
+        total: 1,
+        items: [{ displayValue: "", groupedRawValues: [""] }],
+      });
+      const datePropertyDescription = {
+        name: "#propertyName",
+        displayLabel: "property",
+        typename: "dateTime",
+        editor: undefined,
+      };
+
+      const { queryByText, getByText, user } = render(
+        <UniquePropertyValuesSelector property={datePropertyDescription} onChange={() => {}} imodel={testImodel} descriptor={descriptor} />,
+      );
+
+      // open menu
+      const selector = await waitFor(() => getByText("unique-values-property-editor.select-values"));
+      await user.click(selector);
+
+      // assert that row is displayed correctly
+      await waitFor(() => {
+        expect(queryByText(translate("unique-values-property-editor.empty-value"))).to.not.be.null;
+      });
+    });
+
+    it(`displays date in valid format when typename is 'dateTime'`, async () => {
+      sinon.stub(Presentation.presentation, "getPagedDistinctValues").resolves({
+        total: 1,
+        items: [{ displayValue: "-17655058800", groupedRawValues: [""] }],
+      });
+      const datePropertyDescription = {
+        name: "#propertyName",
+        displayLabel: "property",
+        typename: "dateTime",
+        editor: undefined,
+      };
+
+      const { queryByText, getByText, user } = render(
+        <UniquePropertyValuesSelector property={datePropertyDescription} onChange={() => {}} imodel={testImodel} descriptor={descriptor} />,
+      );
+
+      // open menu
+      const selector = await waitFor(() => getByText("unique-values-property-editor.select-values"));
+      await user.click(selector);
+
+      // assert that row is displayed correctly
+      await waitFor(() => {
+        expect(queryByText(new Date("-17655058800").toLocaleString())).to.not.be.null;
```

Comment:
Running `new Date("-17655058800")` in browser console returns `'Invalid Date'`. Maybe use [ISO 8601 date format](https://en.wikipedia.org/wiki/ISO_8601) something like `1410-07-15T12:34:00Z`

Reply:
Interesting how this works on tests but not in browsers, will adjust